spec/lob/resources/postcard_spec.rb

```diff
@@ -37,12 +37,38 @@
         description: @sample_postcard_params[:description],
         to: new_address["id"],
         message: @sample_postcard_params[:message],
-        front: "https://lob.com/postcardfront.pdf",
+        front: "https://lob.com/postcardfront.pdf"
       )
 
       result["description"].must_equal(@sample_postcard_params[:description])
     end
 
+    it "should accept an idempotency_key" do
```

Comment:
@mgartner @mpiercy827 right, that kind of makes sense, but should we be sending two actual requests to verify their equality for an idempotent behavior? if so, which endpoint do I send it to?